aws_lambda_builders/workflows/custom_make/workflow.py

```diff
@@ -45,17 +44,43 @@ def __init__(self, source_dir, artifacts_dir, scratch_dir, manifest_path, runtim
 
         subprocess_make = SubProcessMake(make_exe=self.binaries["make"].binary_path, osutils=self.os_utils)
 
+        build_in_source = kwargs.get("build_in_source")
```

Comment:
nit: can we move all the kwargs extraction together.

Reply:
done